joern-cli/frontends/rubysrc2cpg/src/test/scala/io/joern/rubysrc2cpg/querying/CallTests.scala

```diff
@@ -567,4 +567,13 @@ class CallTests extends RubyCode2CpgFixture(withPostProcessing = true) {
       case xs => fail(s"Expected two params, got [${xs.code.mkString(",")}]")
     }
   }
+
+  "A Set instantiated with a 'brackets' call" should {
+    val cpg = code("x = Set[]")
+
+    "not be interpreted as an index access" in {
+      cpg.call.nameExact("[]").size shouldBe 1
+      cpg.call.nameExact(Operators.indexAccess).size shouldBe 0
```

Comment:
Please only state our expectation. But in a more precise manner. 
Do we need the `x =` part for the test?
Also check for the `Set` call and then test for the child call to `[]`.

We are not expecting a hole universe of things so only checking for one of those does not make too much sense. So please remove the 'cpg.call.nameExact(Operators.indexAccess).size shouldBe 0'.